chrome_extension/index.css

```diff
@@ -0,0 +1,43 @@
+body {
+    margin: 0;
```

Comment:
please keep one standard in ordering css properties. I.e. width -> margin -> padding -> font-family

https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/